_layouts/menu.html

```diff
@@ -10,10 +10,10 @@
     </button>
     <ul class="submenu">
       <li class="submenu-item">
-        <a class="menu-link" href="https://github.com/tc39/agendas/blob/master/2020/06.md">Upcoming Agenda</a>
+        <a class="menu-link" href="https://github.com/tc39/agendas/blob/master/2020/07.md">Upcoming Agenda</a>
```

Comment:
```suggestion
        <a class="menu-link" href="https://github.com/tc39/agendas/blob/master/2020/09.md">Upcoming Agenda</a>
```
The July meeting happened last week, and I just created the September agenda.